packages/core/src/tracing/instrumentation/protocols/http2Client.js

```diff
@@ -64,13 +64,14 @@ function instrumentClientHttp2Session(clientHttp2Session) {
   const originalRequest = clientHttp2Session.request;
```

Comment:
Putting it here.

issue: there is another usage of getReducedSpan

> const parentSpan = cls.getCurrentSpan(true) || cls.getReducedSpan(true);

In graphql instrumentation.

I assume this won't work if a customer sets "INSTANA_ALLOW_ROOT_EXIT_SPAN=true` and we ignore it in this place.

Reply:
In graphql,
const activeContext = storage.getStore() || (fallbackToSharedContext ? this.sharedContext : undefined);

In the code, since ```parentSpan = cls.getCurrentSpan(true) || cls.getReducedSpan(true);``` true is paased along, we might be using shared context if nothing is found in the current.
  

>    * Retrieves a value by key from the current CLS context (or a parent context), assuming the key/value pair has
>    * been set earlier in this context.
>    *
>    * When `fallbackToSharedContext` is set to true, the caller tries to access a parent context,
>    * which got remembered in a previous instrumentation.

This is somewhat different from other instrumentations as we were looking only at the current CLS context only.

Reply:
Rewrite to be something like:
>   // CASE 3: Third use case is to disable entry spans. If parentSpan is null.